### ISSUE TITLE
How does test step condition work ?

### ISSUE THREAD

**Issue by Thelulucien** [cerberustesting/cerberus-source#1831]
Hi,

I'm quite new to Cerberus, I started using it yesterday, and I don't understand how step condition works.

For instance, the screenshot linked is one of my attempt to make this work.

What am i trying to do here is to close the cookie warning if the element is present in the page.
Why is there two conditions parameters ?

I have read the documentation but it's describing the possible conditions operations, not how to fill fields so I'm a bit lost here.

I know that "IfElementPresent" wait until the timeout error if the element is indeed not there but performance is not an issue (for now..)

![condition_screenshot](https://user-images.githubusercontent.com/17575025/45102951-17762c80-b12f-11e8-9477-3ebb5066b086.png)

Thank you and excuse my English, I'm a baguette eater.

**Comment by vertigo17** [comment]
Hello @Thelulucien and welcome ;-)
What you have put make sense. In fact the ifElementPresent only require 1 parameter. Value2 will be ignored so you can remove 'false'.
Considering that this action is not really part of the test but more to remove the popup in order to perform the rest of the test you could also:
- do a executeJS on a js code that remove that popup (advantage is also that you will have no risk of a FA action)
- work if dev team in order to remove that warning when test is trigered from Cerberus. 
What result do you get from your config ?
NB: This is not Step but Action.

**Comment by Thelulucien** [comment]
Re !

Sorry for the late reply, I did not have to opportunity to this solution until today.

I followed your first recommendation and it worked ! A simple JS script did the trick.
The dev team I'm working with is in a rush now so I did not bother them with a small issue like this so the js workaround is perfectly fine !

Thank you for your quick answer !
I did notice that you are the creator of Cerberus so thanks for your nice testing tool, it always cool to see good french open-source tools. 

Bonne journée !
